### ISSUE TITLE
bgp_fact test fails on LAG configuration

### ISSUE THREAD

**Issue by marian-pritsak** [Azure/sonic-mgmt#136]
```
TASK [test : Compare the bgp neighbors name with minigraph bgp neigbhors name] *
Friday 10 March 2017  11:53:05 +0200 (0:00:03.121)       0:00:17.446 ********** 
fatal: [arc-switch1026-lag]: FAILED! => {"failed": true, "msg": "ERROR! 'dict object' has no attribute u'FC00::12'"}
```

When bgp_fact test is runs on LAG configuration, it fails on IPv6 verification,
I noticed that switch5.xml template for LAG differs from switch2.xml in a way that it has IPv6 BGP peers listed in PeeringSessions.

Looks like bgp_fact test has to be updated.

**Comment by taoyl-ms** [comment]
Solved by PR#138. @marian-pritsak, could you please help verify?

**Comment by daall** [comment]
Fixed by #138
